pymfe/mfe.py

```diff
@@ -395,18 +397,50 @@ def _call_summary_methods(
                 metafeat_names.append(".".join((feature_name, sm_mtd_name)))
                 metafeat_times.append(time_sm)
 
-            if verbose:
+            if verbose >= 2:
                 print("Done.")
 
+        if verbose >= 1:
+            print("\rDone Summarizing {0} feature."
+                  .format(feature_name), end="")
+
         return metafeat_names, metafeat_vals, metafeat_times
 
+    @classmethod
+    def _print_verbose_progress(
```

Comment:
There is still a coverage problem. I think it occurs because you did not a test for this function. Could you please do it?

Reply:
Yes. I've not made any new tests previously because I thought that they were unnecessary for verbose-related methods. However, I changed my mind when the "verbose = 0 still printing messages" happened. Now I've added a couple of tests related to the verbose option.